src/app/components/shared/datetime-formats/datetime/datetime/datetime.component.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { Spectator, createComponentFactory } from "@ngneat/spectator";
+import { SharedModule } from "@shared/shared.module";
+import { MockBawApiModule } from "@baw-api/baw-apiMock.module";
+import { DateTime, IANAZone, Settings, SystemZone } from "luxon";
+import { assertTooltip } from "@test/helpers/html";
+import { DatetimeComponent } from "./datetime.component";
+
+// I have created this interface for TypeScript LSP typing and auto completion
+// it should not be used outside of this file
+interface TestCase {
+  name: string;
+  value: Date | DateTime;
+  expectedText: string;
+  expectedTooltip: string;
+  expectedDateTimeAttribute: string;
+  date?: boolean;
+  time?: boolean;
+}
+
+// by setting the defaultZone to utc when we create DateTime objects from ISO strings
+// it doesn't implicitly localize the DateTime object to another timezone
+// this makes the object produced by the iso string predictable
+// eg. an iso string "2020-10-10T12:12:12Z" will always produce the object
+//     { year: 2020, month: 10, day: 10, hour: 12, minute: 12, second: 12 }
+// we re-mock the test runners timezone to a more complex one when the tests are run
+// however, during test setup it is easier to read the DateTime objects if we create
+// them as if the test runner is in utc+00:00
```

Comment:
i'm not at all convinced this is worthwhile - especially given it does not simulate a real world environment

additionally your tests should be unambiguous in their definitions

Reply:
Agreed, I have removed these from my test declarations